test/WebJobs.Script.Tests.Integration/TestScripts/CSharp/ProxyHttpResponse/response.json

```diff
@@ -0,0 +1,98 @@
+﻿{
+  "Content": {
```

Comment:
This is a big JSON file, and I kept it that way to make sure it covers everything. I think it's cleaner to keep it this way rather than do a JObject inlining in the code. no?